supabase gen types different order on my machine vs github actions

# Bug report

## Describe the bug

Trying to implement CI in Github Actions for verifying types have been generated and committed in git by following [this](https://supabase.com/docs/guides/cli/managing-environments#deploy-a-migration).

I am running into a weird issue where the order of the fields get generated slightly differently between my machine and github actions and it's not clear why. I can use a custom script to deep diff the two but this breaks the documented git diff method. I have verified that both cli versions are 1.12.2

## To Reproduce

On local machine:
    1. supabase link --project-ref $PROD_PROJECT_ID
    2. supabase db remote commit
    3. supabase db reset
    4. supabase gen types typescript --local > ../src/types/supabase/index.ts

On github actions:
```name: CI

on:
  pull_request:
  workflow_dispatch:

jobs:
  test:
    runs-on: ubuntu-latest
    defaults:
      run:
        working-directory: ./backend
    steps:
      - uses: actions/checkout@v3

      - uses: supabase/setup-cli@v1
        with:
          version: 1.12.2

      - name: Start Supabase local development setup
        run: supabase start

      - name: Verify generated types are up-to-date
        run: |
          supabase gen types typescript --local > ../src/types/supabase/index.ts
          cd ../src/types/supabase

          if [ "$(git diff --ignore-space-at-eol index.ts | wc -l)" -gt "0" ]; then
            echo "Detected uncommitted changes after build. See status below:"
            git diff
            exit 1
          fi
```

## Expected behavior

Expect the index.ts type file to be identical in both.

The same issue here, tried many different things but with no luck. On the local machine, I'm using a prettier but the type file is ignored, also tried with no prettier on local env at all, using a prettier on remote - still the same.

Can't figure out a solution.

Run supabase gen types typescript --local > DatabaseDefinitions.ts
Detected uncommitted changes after build. See status below:
diff --git a/DatabaseDefinitions.ts b/DatabaseDefinitions.ts
index e4aa507..a[9](https://github.com/User/project/actions/runs/3468842892/jobs/5795122551#step:5:10)dadd0 [10](https://github.com/User/project/actions/runs/3468842892/jobs/5795122551#step:5:11)0644
--- a/DatabaseDefinitions.ts
+++ b/DatabaseDefinitions.ts
@@ -217,41 +217,44 @@ export interface Database {
       }
       orders: {
         Row: {
-          id: string
           user_id: string
           total: number
+          created_at: string
+          id: string
           payment_method: string
           total_items: number
           cart_id: string
           product_number: number
           total_tickets: number
-          created_at: string
         }
         Insert: {
-          id?: string
           user_id: string
           total: number
+          created_at?: string
+          id?: string
           payment_method: string
           total_items: number
           cart_id: string
           product_number: number
           total_tickets: number
-          created_at?: string
         }
         Update: {
-          id?: string
           user_id?: string
           total?: number
+          created_at?: string
+          id?: string
           payment_method?: string
           total_items?: number
           cart_id?: string
           product_number?: number
           total_tickets?: number
-          created_at?: string
         }
       }
       tickets: {
         Row: {
+          user_id: string
+          product_number: number
+          created_at: string
           id: string
           order_id: string
           tag1: number
@@ -259,[11](https://github.com/User/project/actions/runs/3468842892/jobs/5795122551#step:5:12) +262,11 @@ export interface Database {
           tag3: number
           tag4: number
           tag5: number
-          user_id: string
-          product_number: number
-          created_at: string
         }
         Insert: {
+          user_id: string
+          product_number: number
+          created_at?: string
           id?: string
           order_id: string
           tag1: number
@@ -[27](https://github.com/User/project/actions/runs/3468842892/jobs/5795122551#step:5:28)1,11 +274,11 @@ export interface Database {
           tag3: number
           tag4: number
           tag5: number
-          user_id: string
-          product_number: number
-          created_at?: string
         }
         Update: {
+          user_id?: string
+          product_number?: number
+          created_at?: string
           id?: string
           order_id?: string
           tag1?: number
@@ -[28](https://github.com/User/project/actions/runs/3468842892/jobs/5795122551#step:5:29)3,9 +286,6 @@ export interface Database {
           tag3?: number
           tag4?: number
           tag5?: number
-          user_id?: string
-          product_number?: number
-          created_at?: string
         }
       }
     }
Error: Process completed with exit code 1.

--- a/internal/utils/misc.go
+++ b/internal/utils/misc.go
@@ -34,7 +34,7 @@ const (
 	PostgrestImage  = "postgrest/postgrest:v10.1.1.20221215"
 	DifferImage     = "supabase/pgadmin-schema-diff:cli-0.0.5"
 	MigraImage      = "djrobstep/migra:3.0.1621480950"
-	PgmetaImage     = "supabase/postgres-meta:v0.53.1"
+	PgmetaImage     = "supabase/postgres-meta:v0.54.1"
 	StudioImage     = "supabase/studio:20221214-4eecc99"
 	DenoRelayImage  = "supabase/deno-relay:v1.5.0"
 	ImageProxyImage = "supabase/imgproxy:v1.0.4"

--- a/internal/utils/templates/initial_schemas/15.sql
+++ b/internal/utils/templates/initial_schemas/15.sql
@@ -1380,6 +1380,7 @@ ALTER TABLE ONLY auth.refresh_tokens ALTER COLUMN id SET DEFAULT nextval('auth.r
 -- Data for Name: extensions; Type: TABLE DATA; Schema: _realtime; Owner: postgres
 --
 
+INSERT INTO _realtime.extensions (id, type, settings, tenant_external_id, inserted_at, updated_at) VALUES ('935b3773-e37c-4b3e-baa1-2309ddb697cf', 'postgres_cdc_rls', '{"region": "us-east-1", "db_host": "I4YigNuoYSZbe+Xs4vInbw==", "db_name": "sWBpZNdjggEPTQVlI52Zfw==", "db_port": "MqmbZ5ZiXXFlSy8FeFYPAQ==", "db_user": "sWBpZNdjggEPTQVlI52Zfw==", "slot_name": "supabase_realtime_replication_slot", "ip_version": 4, "db_password": "sWBpZNdjggEPTQVlI52Zfw==", "publication": "supabase_realtime", "poll_interval_ms": 100, "poll_max_changes": 100, "poll_max_record_bytes": 1048576}', 'realtime-demo', '2022-11-14 23:04:49', '2022-11-14 23:04:49');
 
 
 --
@@ -1402,6 +1403,7 @@ INSERT INTO _realtime.schema_migrations (version, inserted_at) VALUES (202211021
 -- Data for Name: tenants; Type: TABLE DATA; Schema: _realtime; Owner: postgres
 --
 
+INSERT INTO _realtime.tenants (id, name, external_id, jwt_secret, max_concurrent_users, inserted_at, updated_at, max_events_per_second, postgres_cdc_default) VALUES ('98d09d90-612d-4a69-8c84-e8e5ecbaed6d', 'realtime-demo', 'realtime-demo', 'iNjicxc4+llvc9wovDvqymwfnj9teWMlyOIbJ8Fh6j2WNU8CIJ2ZgjR6MUIKqSmeDmvpsKLsZ9jgXJmQPpwL8w==', 300, '2022-11-14 23:04:49', '2022-11-14 23:04:49', 100, 'postgres_cdc_rls');
 
 
 --
@@ -1496,22 +1498,6 @@ INSERT INTO auth.schema_migrations (version) VALUES ('20221027105023');
 
 
 --
--- Data for Name: tenants; Type: TABLE DATA; Schema: _realtime; Owner: supabase_admin
---
-
-INSERT INTO _realtime.tenants (id, name, external_id, jwt_secret, max_concurrent_users, max_events_per_second, postgres_cdc_default, inserted_at, updated_at)
-    VALUES ('98d09d90-612d-4a69-8c84-e8e5ecbaed6d', 'realtime-demo', 'realtime-demo', 'iNjicxc4+llvc9wovDvqymwfnj9teWMlyOIbJ8Fh6j2WNU8CIJ2ZgjR6MUIKqSmeDmvpsKLsZ9jgXJmQPpwL8w==', 300, 100, 'postgres_cdc_rls', '2022-11-14 23:04:49', '2022-11-14 23:04:49');
-
-
---
--- Data for Name: extensions; Type: TABLE DATA; Schema: _realtime; Owner: supabase_admin
---
-
-INSERT INTO _realtime.extensions (id, settings, type, tenant_external_id, inserted_at, updated_at)
-    VALUES ('935b3773-e37c-4b3e-baa1-2309ddb697cf', '{"region":"us-east-1","db_host":"I4YigNuoYSZbe+Xs4vInbw==","db_name":"sWBpZNdjggEPTQVlI52Zfw==","db_port":"MqmbZ5ZiXXFlSy8FeFYPAQ==","db_user":"sWBpZNdjggEPTQVlI52Zfw==","slot_name":"supabase_realtime_replication_slot","ip_version":4,"db_password":"sWBpZNdjggEPTQVlI52Zfw==","publication":"supabase_realtime","poll_interval_ms":100,"poll_max_changes":100,"poll_max_record_bytes":1048576}', 'postgres_cdc_rls', 'realtime-demo', '2022-11-14 23:04:49', '2022-11-14 23:04:49');
-
-
---
 -- Data for Name: sessions; Type: TABLE DATA; Schema: auth; Owner: supabase_auth_admin
 --
 
@@ -1539,13 +1525,6 @@ INSERT INTO _realtime.extensions (id, settings, type, tenant_external_id, insert
 -- Data for Name: users; Type: TABLE DATA; Schema: auth; Owner: supabase_auth_admin
 --
 
-
-
---
--- Data for Name: key; Type: TABLE DATA; Schema: pgsodium; Owner: postgres
---
-
--- INSERT INTO pgsodium.key (id, status, created, expires, key_type, key_id, key_context, name, associated_data, raw_key, raw_key_nonce, parent_key, comment) VALUES ('463342b1-c264-4850-931e-9afa154f151d', 'valid', '2022-12-14 17:16:50.253224+00', NULL, 'aead-det', 1, '\x7067736f6469756d', 'default_vault_key', '', NULL, NULL, NULL, NULL);
 
 
 --

